test/unit/auth/auth-api-request.spec.ts

```diff
@@ -4448,7 +4453,7 @@ AUTH_REQUEST_HANDLER_TESTS.forEach((handler) => {
       describe('createTenant', () => {
         const path = '/v2/projects/project_id/tenants';
         const postMethod = 'POST';
-        const tenantOptions: TenantOptions = {
+        const tenantOptions: UpdateTenantRequest = {
```

Comment:
I think this should be `CreateTenantRequest`

Reply:
Done